google/cloud/forseti/common/gcp_type/firewall_rule.py

```diff
@@ -511,15 +511,20 @@ def __lt__(self, other):
           bool: comparison result
         """
         LOGGER.debug('Checking %s < %s', self, other)
-        return ((self.direction == other.direction or
-                 self.direction is None or
-                 other.direction is None) and
-                (self.network == other.network or other.network is None) and
-                set(self.source_tags).issubset(other.source_tags) and
-                set(self.target_tags).issubset(other.target_tags) and
-                self.firewall_action < other.firewall_action and
-                ips_in_list(self.source_ranges, other.source_ranges) and
-                ips_in_list(self.destination_ranges, other.destination_ranges))
+        direction = (self.direction == other.direction or
+                     self.direction is None or
+                     other.direction is None)
+        network = (self.network == other.network or
+                   other.network is None)
+        source_tags = set(self.source_tags).issubset(other.source_tags)
+        target_tags = set(self.target_tags).issubset(other.target_tags)
+        firewall_action = self.firewall_action < other.firewall_action
+        source_ranges = ips_in_list(self.source_ranges, other.source_ranges)
+        destination_ranges = ips_in_list(self.destination_ranges,
+                                         other.destination_ranges)
+        result = (direction and network and source_tags and target_tags
```

Comment:
Can you make this more readable by organizing them in a single line?  Can you please make this change everywhere?
```
result = (direction and
          network and
          source_tags and
          target_tags)
```

Reply:
Done